.travis.yml

```diff
@@ -27,28 +27,14 @@ before_install:
     fi
 
 install:
-  # beheve is a dev dependeny, so not in requirements.txt.
-  - pip install behave
   - pip install -r ./requirements.txt
-  # codecov is a code coverage tool.
   - pip install codecov
-  - cd ./vscode-client
-  - npm install
-  - mkdir ./out
-  - npm run vscode:prepublish
   - cd - > /dev/null
 
 script:
   - >
-  # https://github.com/coala/coala-bears/issues/1037
   - sed -i.bak '/bears = GitCommitBear/d' .coafile
-  # Server side tests.
-  - coverage run $(which behave) ./tests/server.features
-  - coverage run -a -m unittest discover -s tests
-  # Frontend tests.
-#  - cd ./vscode-client
-#  - npm test
-#  - cd - > /dev/null
+  # - coverage run -a -m unittest discover -s tests
```

Comment:
This PR would be a regression.

The test changes need to be part of this PR, so it isnt a regression.

but before that, need an answer at https://github.com/coala/coala-ls/pull/3/files#r193274414